shared/chat/conversation/block-conversation-warning/container.js

```diff
@@ -22,16 +22,20 @@ const mapStateToProps = (state: TypedState, {routeProps}: OwnProps) => {
 }
 
 const mapDispatchToProps = (dispatch: Dispatch) => ({
-  onBlock: (conversationIDKey: ConversationIDKey) => dispatch(({payload: {blocked: true, conversationIDKey}, type: 'chat:blockConversation'}: BlockConversation)),
+  onBlock: (conversationIDKey: ConversationIDKey, reportUser: boolean) => dispatch(({payload: {blocked: true, conversationIDKey, reportUser}, type: 'chat:blockConversation'}: BlockConversation)),
   onBack: () => dispatch(navigateUp()),
 })
 
 const mergeProps = (stateProps, dispatchProps, ownProps) => ({
   ...stateProps,
   ...dispatchProps,
   ...ownProps,
+  onBlockAndReport: () => {
+    dispatchProps.onBlock(stateProps.conversationIDKey, true)
+    dispatchProps.onBack()
+  },
   onBlock: () => {
-    dispatchProps.onBlock(stateProps.conversationIDKey)
+    dispatchProps.onBlock(stateProps.conversationIDKey, false)
```

Comment:
:sheep: up to you, but perhaps consider switching from positional to object args now that there's more than one arg